src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -28,7 +28,7 @@ End Class
                 Await state.AssertSelectedCompletionItem("On Error GoTo", description:=String.Format(FeaturesResources.Keyword, "On Error GoTo") + vbCrLf + VBFeaturesResources.OnErrorGotoKeywordToolTip)
                 state.SendTypeChars(" ")
                 Await state.WaitForAsynchronousOperationsAsync()
-                Await state.AssertSelectedCompletionItem("Error GoTo", description:=String.Format(FeaturesResources.Keyword, "Error GoTo") + vbCrLf + VBFeaturesResources.OnErrorGotoKeywordToolTip)
+                Await state.AssertSelectedCompletionItem("On Error GoTo", description:=String.Format(FeaturesResources.Keyword, "On Error GoTo") + vbCrLf + VBFeaturesResources.OnErrorGotoKeywordToolTip)
```

Comment:
Note: this is "regression" that i think is ok.  Namely, because 'on ' matches the text in the completion item, we say "continue filtering and don't commit".  I think this is personally ok because the end experience is the same.  With this approach we also don't have the completion list comitting then popping up immediately with that single item.